src/desktop/apps/artwork2/client.tsx

```diff
@@ -103,10 +103,12 @@ mediator.on("openViewInRoom", options => {
       newHeight = newWidth
     }
 
+    const newLeft = (bounds.width - newWidth) / 2 + bounds.left
+
     const positionStyles = {
       position: "absolute",
       top: `${bounds.top + Math.abs(bounds.height - newHeight) / 2}px`,
-      left: `${bounds.left}px`,
+      left: `${newLeft}px`,
```

Comment:
:+1:

one very minor non-blocking suggestion I'd make would be to move the definition for `top` to a `newTop` const right above the `newLeft` const to show that top and left are related.